We remove source maps from the production build for three reasons:
- They have no reason to even be in there
- They reduce the size of the production build
- They contain references to a temporary build dir, making the build non reproducible

--- a/app/build/tasks/package-task.js	2026-04-02 11:53:06.912377645 +0200
+++ b/app/build/tasks/package-task.js	2026-04-02 11:53:39.343655349 +0200
@@ -102,7 +102,15 @@
         if (tsPath.endsWith('.d.ts')) return;
         const outPath = tsPath.replace(path.extname(tsPath), '.js');
         console.log(`  ---> Compiling ${tsPath.slice(tsPath.indexOf('/app') + 4)}`);
-        const res = TypeScript.transpileModule(tsCode, { compilerOptions, fileName: tsPath });
+        const res = TypeScript.transpileModule(tsCode, {
+            compilerOptions: {
+                ...compilerOptions,
+                sourceMap: false,
+                inlineSourceMap: false,
+                inlineSources: false,
+            },
+            fileName: tsPath,
+            });
         grunt.file.write(outPath, res.outputText);
         fs.unlinkSync(tsPath);
       });
